### PR TITLE
CB-1095 Add redbeams

### PR DESCRIPTION
**Pull request by bhavanki**
The deployer now has the ability to manage a container running the
new Redbeams service, along with its database.

### CODE REVIEW

File: templates/compose-redbeams.tmpl [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            - REDBEAMS_HOSTNAME_RESOLUTION=public
+            - REDBEAMS_ADDRESS_RESOLVING_TIMEOUT
+            - "REDBEAMS_CLOUDBREAK_URL={{{get . "CLOUDBREAK_URL"}}}"
+            - REDBEAMS_SCHEMA_SCRIPTS_LOCATION
+            - REDBEAMS_SCHEMA_MIGRATION_AUTO
+            - REDBEAMS_INSTANCE_NODE_ID={{{get . "CB_INSTANCE_NODE_ID"}}}
+            - REDBEAMS_LOG_LEVEL
+        labels:
+            - traefik.port=8080
+            - traefik.frontend.rule=PathPrefix:/rdb/
```
Comment (jadair-cloudera) [inline templates/compose-redbeams.tmpl]: rdb or redbeams for the prefix?

File: traefik_test.go [hunk 2]
```diff
@@ -52,11 +56,16 @@ var expectedMulti string = `[file]
         [frontends.periscope-frontend.routes.frontendrule]
         rule = "PathPrefix:/as/"
         priority = 100
+    [frontends.redbeams-frontend]
+    backend = "redbeams"
+        [frontends.redbeams-frontend.routes.frontendrule]
+        rule = "PathPrefix:/rdb/"
```
Comment (jadair-cloudera) [inline traefik_test.go]: rdb or redbeams?

File: include/migrate.bash [hunk 3]
```diff
@@ -123,30 +129,36 @@ execute-migration() {
...
                     if [[ "$CB_SCHEMA_SCRIPTS_LOCATION" == "container" && "$CB_LOCAL_DEV_LIST" == *"cloudbreak"* ]]; then
-                        migrateError "CB_SCHEMA_SCRIPTS_LOCATION environment variable must be set and points to the cloudbreak project's schema location"
+                        migrateError "CB_SCHEMA_SCRIPTS_LOCATION environment variable must be set and pointing to the cloudbreak project's schema location"
                         _exit 127
                     fi
                     ;;
                 periscopedb)
                     if [[ "$PERISCOPE_SCHEMA_SCRIPTS_LOCATION" == "container" && "$CB_LOCAL_DEV_LIST" == *"periscope"* ]]; then
-                        migrateError "PERISCOPE_SCHEMA_SCRIPTS_LOCATION environment variable must be set and points to the autoscale project's schema location"
+                        migrateError "PERISCOPE_SCHEMA_SCRIPTS_LOCATION environment variable must be set and pointing to the autoscale project's schema location"
```
[the remark this answers is not in the document]
Comment (bhavanki) [reply include/migrate.bash]: Not sure - my only reason for changing this file was the grammar :)

File: templates/compose-redbeams.tmpl (hunk 1 above)
Comment (bhavanki) [reply templates/compose-redbeams.tmpl]: I'm shooting for "rdb". The other services have very short prefixes as well.

File: include/env.bash [hunk 4]
```diff
@@ -175,6 +177,16 @@ PERISCOPE_DB_PORT_5432_TCP_PORT - Port number of the Autoscale database
...
+REDBEAMS_DB_ENV_DB - Name of the Redbeams database
+REDBEAMS_DB_ENV_PASS - Password for the Redbeams database authentication
+REDBEAMS_DB_ENV_SCHEMA - Used schema in the Redbeams database
+REDBEAMS_DB_ENV_USER - User for the Redbeams database authentication
+REDBEAMS_DB_PORT_5432_TCP_ADDR - Address of the Redbeams database
+REDBEAMS_DB_PORT_5432_TCP_PORT - Port number of the Redbeams database
+REDBEAMS_LOG_LEVEL - Log level of the Redbeams service
+REDBEAMS_SCHEMA_MIGRATION_AUTO - Flag for Redbeams automatic database schema update
+REDBEAMS_SCHEMA_SCRIPTS_LOCATION - Location of Redbeams schema update files
 REST_DEBUG - Enables REST call debug level in Cloudbreak and Autoscale
```
[the remark this answers is not in the document]
Comment (bhavanki) [reply include/env.bash]: Indeed it does

File: include/migrate.bash [hunk 5]
```diff
@@ -123,30 +129,36 @@ execute-migration() {
...
                 datalakedb)
                     if [[ "$DATALAKE_SCHEMA_SCRIPTS_LOCATION" == "container" && "$CB_LOCAL_DEV_LIST" == *"datalake"* ]]; then
-                        migrateError "DATALAKE_SCHEMA_SCRIPTS_LOCATION environment variable must be set and points to the autoscale project's schema location"
+                        migrateError "DATALAKE_SCHEMA_SCRIPTS_LOCATION environment variable must be set and pointing to the autoscale project's schema location"
+                        _exit 127
+                    fi
+                    ;;
+                redbeamsdb)
+                    if [[ "$REDBEAMS_SCHEMA_SCRIPTS_LOCATION" == "container" && "$CB_LOCAL_DEV_LIST" == *"redbeams"* ]]; then
+                        migrateError "REDBEAMS_SCHEMA_SCRIPTS_LOCATION environment variable must be set and pointing to the Redbeams project's schema location"
```
[the remark this answers is not in the document]
Comment (bhavanki) [reply include/migrate.bash]: fixed

File: traefik_test.go (hunk 2 above)
Comment (bhavanki) [reply traefik_test.go]: rdb, see previous comment

**Pull request merged**